ramses_inspector.py

```diff
@@ -22,28 +23,30 @@ def __init__(self,
                  scene: ExportableScene):
 
         self.scene = scene
-        self._window = None
+        self.viewer_process = None
 
-    def load_viewer(self):
+    def load_viewer(self, platform, block: bool = False):
         """Loads the RAMSES scene viewer for visual inspection of the
         generated scene"""
 
+        assert isinstance(platform, str)
+        platform = platform.lower()
```

Comment:
why not put it to lower in the list in the first place?

Reply:
Reworked this in 01808a7